gremlin-test/src/main/java/org/apache/tinkerpop/gremlin/structure/VertexTest.java

```diff
@@ -280,10 +280,9 @@ public void shouldHaveExceptionConsistencyWhenIdNotSupportedForAddEdge() throws
         }
 
         @Test
-        @FeatureRequirementSet(FeatureRequirementSet.Package.VERTICES_ONLY)
-        @FeatureRequirement(featureClass = VertexPropertyFeatures.class, feature = FEATURE_INTEGER_VALUES)
+        @FeatureRequirement(featureClass = Graph.Features.VertexFeatures.class, feature = Graph.Features.VertexFeatures.FEATURE_ADD_VERTICES)
         public void shouldHaveStandardStringRepresentation() {
-            final Vertex v = graph.addVertex("name", "marko", "age", 34);
+            final Vertex v = graph.addVertex();
```

Comment:
This might be changing the semantics of the test a little. We want to validate that a vertex looks a certain way with `toString()`. By adding properties, it introduces some data to the vertex object and allows the test to validate that none of that state shows up in the `toString()`.  Maybe an off-chance of that happening I suppose. I guess the benefit is that we widen the test footprint for providers who don't support adding properties (but that's a really super basic thing). is there some other benefit? wdyt?